articles/macos-mdm-setup.md

```diff
@@ -4,7 +4,9 @@ To turn on macOS, iOS, and iPadOS MDM features, follow the instructions on this
 
 To use automatic enrollment (aka zero-touch) features on macOS, iOS, and iPadOS, follow instructions to connect Fleet with Apple Business Manager (ABM).
 
-To turn on Windows MDM features, head to this [Windows MDM setup article](https://fleetdm.com/guides/windows-mdm-setup). 
+To turn on Windows MDM features, head to this [Windows MDM setup article](https://fleetdm.com/guides/windows-mdm-setup).
+
+> Fleet automatically renews the SCEP certificate 180 days before the expiration date.
```

Comment:
What is SCEP? What does Fleet/the host use it for? 

I think we want to be explicit (spell out the acronym) so that anyone new to IT can understand.

Also, since SCEP is something that happens behind the scenes I think we can move this item down to the bottom of the guide. Maybe add a new SCEP section?

Reply:
@noahtalerman I'm not comfortable writing technical details of what is SCEP and how it works. I guess we could use engineering help here.

I agree we could add it to the bottom. I think the main use case of this section will be if someone is Googling or searching in docs.

Reply:
No worries @marko-lisica!

I committed on top of this PR. Let me know what you think.

I used the SCEP definition in our glossary: https://github.com/fleetdm/fleet/blob/main/tools/mdm/apple/glossary-and-protocols.md#scep-simple-certificate-enrollment-protocol

Reply:
@noahtalerman Thanks, LGTM. I forgot to check our glossary.